### ISSUE TITLE
CMake FindOpenSubdiv module does not recognize RC builds

### ISSUE THREAD

**Issue by jcowles** [PixarAnimationStudios/USD#82]
### Steps to Reproduce
1. Build OpenSubdiv RC2
2. Run CMake configuration for USD

Note that OpenSubdiv is recognized as version "2" (vs. 3.1.0.RC2).

This is due to the regex in the CMake config, I have a local fix if there is interest in making the RC builds work.

**Comment by jtran56** [comment]
Filed as internal issue #138795.

**Comment by c64kernal** [comment]
Thanks Jeremy.  We have a fix coming to dev soon.

**Comment by meshula** [comment]
@jcowles could you summarize your fix briefly here? I've got a couple of questions about this already.

**Comment by c64kernal** [comment]
Hey Nick and Jeremy -- the fix is now in dev.  Let us know if you have any questions.

**Comment by jcowles** [comment]
Thanks for the quick fix!

@meshula I basically did what takahito did, but I chopped off _RC* instead.

**Comment by meshula** [comment]
Thanks :)
